provisioning/transport/amqp/tests/ProvisioningErrorDetailsAmqpTests.cs

```diff
@@ -0,0 +1,149 @@
+﻿using Microsoft.Azure.Amqp;
+using Microsoft.Azure.Amqp.Encoding;
+using Microsoft.Azure.Amqp.Framing;
+using Microsoft.Azure.Devices.Provisioning.Client.Transport;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Devices.Provisioning.Transport.Amqp.UnitTests
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class ProvisioningErrorDetailsAmqpTests
+    {
+        private static TimeSpan defaultInterval = TimeSpan.FromSeconds(2);
+
+        [TestMethod]
+        public void getRetryFromRejectedSuccess()
```

Comment:
Please follow our naming convention for tests <Unit>_<Scenario>_<ExpectedResult>

Reply:
I'll make them start with capital letters, sure